tests/test_bgp_af.rb

```diff
@@ -1081,4 +1141,98 @@ def route_target_tester(bgp_af, af, opts, should, test_id)
                    "#{test_id} : #{af} : route_target_#{opt}_evpn")
     end
   end
+
+  ##
+  ## suppress_inactive
+  ##
+  def suppress_inactive(asn, vrf, af)
+    bgp_af = RouterBgpAF.new(asn, vrf, af)
+
+    #
+    # Set and verify
+    #
+    val = false
+    bgp_af.suppress_inactive = val
+    assert_equal(val, bgp_af.suppress_inactive,
```

Comment:
Don't use `assert_equal` for bool tests -- use `refute` when testing for false, `assert` for true.